src/api/userActionTime/services.py

```diff
@@ -13,6 +13,8 @@
 from sqlalchemy import func, literal_column, and_
 
 def create_or_update_user_action_time(date: str, duration: float, id_user: int, id_action: int):  
+    if duration >= 0 or duration <= 24:
```

Comment:
you are always raising an error, because duration can't be >0 and <24 at the same time,
```suggestion
    if duration < 0 or duration > 24:
```